QuizApp/App/Screens/Quiz/QuizViewModel.swift

```diff
@@ -0,0 +1,15 @@
+import Foundation
+
+final class QuizViewModel {
+
+    var quizCategories: [Category] {
```

Comment:
Create a separate class named Category and map from CategoryNetworkModel -> Category.